src/RemoteStorageService/Confirmation.test.js

```diff
@@ -0,0 +1,23 @@
+import '../../test/jest/__mock__';
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { Heading, Message } from './Confirmation';
+
+describe('Confirmation', () => {
+  it('should render remote confirmation message', () => {
+    const { getByText } = render(
+      <IntlProvider>
+        <Heading />
+      </IntlProvider>
+    );
+    expect(getByText('ui-inventory.remote.confirmation.heading')).toBeInTheDocument();
+  });
+  it('should render remote items message', () => {
+    render(
+      <IntlProvider>
+        <Message />
+      </IntlProvider>
+    );
+    expect(screen.getByText('ui-inventory.remote.confirmation.message')).toBeInTheDocument();
+  });
```

Comment:
Thinking through this a little more, there should really be two tests here: 

1. without a `count` prop, Message renders the expected default value
2. with a `count` prop, Message renders the given value

Reply:
Zak,
We made changes as you suggested
Please review and let me know if any.